src/main/groovy/org/pih/warehouse/api/StockMovement.groovy

```diff
@@ -471,34 +475,17 @@ class StockMovement implements Validateable{
         ]
     }
 
-    static def getApplicationTagLib() {
-        return Holders.grailsApplication.mainContext.getBean('org.grails.plugins.web.taglib.ApplicationTagLib')
-    }
-
-    // Because this function needs to handle more than one enum, I left the argument's type as def
-    String getTranslatedDisplayStatus(def status) {
-        return applicationTagLib.message(code: 'enum.' + status?.class?.simpleName + '.' + status)
-    }
-
-    // Mapping statuses for display for the requestor's dashboard
-    // We want to display all statuses from approval workflow (approved, rejected and pending approval)
-    // We want to map all statuses from depot side to "in progress".
-    String getDisplayStatus() {
-        switch(requisition?.status) {
-            case RequisitionStatus.APPROVED:
-                return getTranslatedDisplayStatus(StockMovementStatusCode.APPROVED)
-            case RequisitionStatus.REJECTED:
-                return getTranslatedDisplayStatus(StockMovementStatusCode.REJECTED)
-            case RequisitionStatus.PENDING_APPROVAL:
-                return getTranslatedDisplayStatus(StockMovementStatusCode.PENDING_APPROVAL)
-            case RequisitionStatus.VERIFYING:
-            case RequisitionStatus.PICKING:
-            case RequisitionStatus.PICKED:
-            case RequisitionStatus.CHECKING:
-                return getTranslatedDisplayStatus(StockMovementStatusCode.IN_PROGRESS)
-            default:
-                return getTranslatedDisplayStatus(shipment?.status?.code)
-        }
+    @Deprecated
```

Comment:
same here. Please document why we're deprecating and what we should use instead.